draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -109,18 +109,15 @@ Host: example.org
 </artwork>
 
     <t>
-      if the query parameters extend to several kilobytes or more of
-      data it may not be, because many implementations place limits on their size. Often these limits are not
-      known or discoverable ahead of time, because a request can pass through many uncoordinated
-      systems. Additionally, expressing certain kinds of data in the target URI is inefficient, because it
-      needs to be encoded to be a valid URI.
-    </t>
-
-    <t>
-      Encoding query parameters directly into the request URI also effectively
-      casts every possible combination of query inputs as distinct
-      resources. Depending on the application, that may not be desirable.
+      However, for a query with parameters that are complex or large in size,
+      encoding it in the request URI may not be the best option because
     </t>
+      <ul>
+        <li>often size limits are not known ahead of time,</li>
```

Comment:
```suggestion
        <li>often size limits are not known ahead of time because a request can pass through many uncoordinated
      system,</li>
```